### ISSUE TITLE
Building block: block_check_checksum

### ISSUE THREAD

**Issue by r06942072** [NAL-i5K/Organism_Onboarding#7]
Link: https://github.com/NAL-i5K/CWL_Common-Workflow-Language/tree/dev/block_check_checksum

equivalent command : (--check = -c)
```
md5sum -c md5checksums.txt &> md5checksums.log
md5sum -c md5checksums-fake.txt &> md5checksums-fake.log
```
To demo the fail of check md5sum, I purposely modify the hash of one downloaded file.
The key difference between **md5checksums.log** and **md5checksums-fake.log** is the message :
"checksums did NOT match"

(e.g.)
Here is the example result.
Downloading 4 .gz files and a .txt file (It works if we only download one .txt file)
We find the .txt file and then run **md5sum -c [nameroot].txt**

**md5checksums.log**
[https://app.zenhub.com/files/153128643/8cbcb812-cb3c-464d-bcfa-d8e037a4a5bd/download](https://app.zenhub.com/files/153128643/8cbcb812-cb3c-464d-bcfa-d8e037a4a5bd/download)

**md5checksums-fake.log**
[https://app.zenhub.com/files/153128643/7261f2d2-2316-42e6-b40d-57dbb8f99334/download](https://app.zenhub.com/files/153128643/7261f2d2-2316-42e6-b40d-57dbb8f99334/download)

After block_check_checksum, I would like to design a **grep** block to detect a key sentence in warning message, which is written in check.log file.

In other words, to design a block with equivalent command :
```
grep 'checksums did NOT match' md5checksums.log
```

**Comment by r06942072** [comment]
$ tree -C
├── data
│   ├── GCF_002532875.1_vjacob_1.0_genomic.fna.gz
│   ├── GCF_002532875.1_vjacob_1.0_genomic.gff.gz
│   ├── GCF_002532875.1_vjacob_1.0_rna_from_genomic.fna.gz
│   ├── GCF_002532875.1_vjacob_1.0_translated_cds.faa.gz
│   ├── md5checksums-fake.txt
│   └── md5checksums.txt
├── md5sum_check.cwl
├── md5sum_check-job.yml

If the "data" directory is one I put all the <.gz> files and the md5checksums.txt
Then a possible way to do is to design a command like this : 
```
cd data && md5sum -c md5checksums-fake.txt &> check-fake.log
```

**Comment by r06942072** [comment]
We extract what we have downloaded to another new file 
md5checksums.txt2
